src/isomorphic/deprecated/cloneWithProps.js

```diff
@@ -28,10 +28,16 @@ var CHILDREN_PROP = keyOf({children: null});
  * @param {object} props props you'd like to modify. className and style will be
  * merged automatically.
  * @return {ReactElement} a clone of child with props merged in.
+ * @deprecated
  */
 function cloneWithProps(child, props) {
   if (__DEV__) {
     warning(
+      false,
```

Comment:
Do we want to warn on every call? Typically we try to scope this so that we don't warn on every call to render, which could be really noisy. In some cases we scope to the parent/owner, others to a single deprecation overall.